### PR TITLE
✨📝Feat: Created Home Page News Section Structure

### LINKED ISSUE #7: 🎨 Create Home Page News Section Component
#  🎨 Create the Home Page News Section Component
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that displays all news- items(blogs) that I have on my homepage.

NewsPage

Please take note of all the design features

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```

### Technical Analysis

This component will be created in `index.html`

### Acceptance Criteria

- [ ] When rendered the section is pixel-perfect compared to the design above.
- [ ] The section is aligned using CSS grid
- [ ] The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by manotim**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #7 

## Created Home Page for News Section

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
- [x] Included in the thread below

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -3,3 +3,71 @@
...
+
+h1 {
+  font-size: 42px;
+  /* font-family:"Times New Roman", serif; */
+  font-family: 'Onest', sans-serif;
+  font-weight: 600;
+  color: #202020;
+  margin-left: 120px;
+  margin-bottom: 100px;
+}
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use a single margin property  example margin: 10px 10px; specifies margin top and bottom as 10px and right and left as 10px,

File: src/css/style.css [hunk 2]
```diff
@@ -3,3 +3,71 @@
...
+  display: flex;
+  flex-direction: row; /* Arrange children in a row */
+  justify-content: space-between; /* Distribute space between children */
+  align-items: center; /* Center items vertically */
+  border-bottom: 2px solid #b8b7b7; /* Add a border to each article */
+  /* box-shadow: 0px 10px 10px rgba(0, 0, 0, 0.1); */
+  padding: 10px; /* Add padding to create spacing between content and border */
+  margin-left: 110px;
+  margin-right: 100px;
+  margin-top: 50px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: use single margin

File: src/css/style.css [hunk 3]
```diff
@@ -3,3 +3,71 @@
...
+  filter: blur(0.7px); /* Adjust the blur intensity as needed */
+  color: #5f5c5c; /* Change the color to grayish */
+  font-size: 15px;
+  margin-bottom: 45px;
+}
+
+section {
+  margin-top: 140px;
+  margin-left: 50px;
+  margin-right: 100px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css]: same here

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
please attach screenshots of web and mobile views

**Comment by manotim** [comment]
Screenshot from 2023-10-03 13-22-28
Screenshot from 2023-10-03 13-31-47

File: src/css/style.css (hunk 2 above)
Comment (manotim) [reply src/css/style.css]: Done

File: src/css/style.css (hunk 3 above)
Comment (manotim) [reply src/css/style.css]: Done

**Comment by manotim** [comment]
Screenshots for web and mobile view now attached.

**Comment by manotim** [comment]
Done. Kindly, Review.

**Comment by JAPHETHNYARANGA** [comment]
Thia is good work, ATleast you are learning. Closing this pr as it is already done

**Pull request closed without merging**